docs/registry/_index.md

```diff
@@ -27,18 +27,33 @@ aliases:
 menuindent: true
 ---
 
+The [Viam registry](https://app.viam.com/registry) is the central place where you can browse [ML models](/ml) to deploy with machine applications like [computer vision](/ml/vision/), or {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`.
```

Comment:
I think this will avoid people missing the second part of the sentence when skimming over it
```suggestion
The [Viam registry](https://app.viam.com/registry) is the central place where you can browse:

 - [ML models](/ml) to deploy with machine applications like [computer vision](/ml/vision/)
 - {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`
```